zulipterminal/ui_tools/views.py

```diff
@@ -616,6 +616,7 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
                 recipient_emails=[email],
                 contextual_message_id=pm,
             )
+            return key
```

Comment:
This change is small, but I split it out from the first commit since it's a distraction from the main refactor.